core/tests/protractor_utils/general.js

```diff
@@ -41,6 +41,17 @@ var checkForConsoleErrors = function(errorsToIgnore) {
         return !(browserLog.message.includes(' Slow network is detected.'));
       });
     }
+
+    // Since we are in the process of upgrading CKeditor-4 to Ckeditor-5,
+    // we are observing consoles errors for duplicated modules
+    // (for more details,
+    // please refer to https://github.com/ckeditor/ckeditor5/issues/1821)
+    // and therefore, we need to filter such logs until we've upgraded to
+    // CKEditor-5 completely.
+    browserLogs = browserLogs.filter(function(browserLog) {
```

Comment:
Can we also file an issue for this and assign it to you? Else we might forget about this.

Reply:
Missed now updated PTAL!

Reply:
Done Added here #7224 Thanks!